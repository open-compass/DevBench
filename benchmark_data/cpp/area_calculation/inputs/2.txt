3
4 6
100