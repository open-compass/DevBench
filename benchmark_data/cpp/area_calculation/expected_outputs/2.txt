Shape()
Circle(r)
28.2743
Shape()
Rectangle(w, h)
24
Shape()
Rectangle(w, h)
Square(l)
10000
~Square()
~Rectangle()
~Shape()
~Rectangle()
~Shape()
~Circle()
~Shape()
